### PR TITLE
Category badges added 

### PR DESCRIPTION
**Pull request by Mathursan-007**
issue #1 
- Category theme is not found.
- All category tags were added to the bottom of the card
image

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
Hey Can you check this  [page](https://designrevision.com/demo/shards-dashboard-lite-react/blog-posts) 
I expected like the below screenshot
Screenshot from 2021-11-25 23-12-20.
 you can download and play with this [repo](https://designrevision.com/downloads/shards-dashboard-lite-react/).. 
Don't worry about the styling. It is already added in the assert file.

**Comment by Mathursan-007** [comment]
> Hey Can you check this [page](https://designrevision.com/demo/shards-dashboard-lite-react/blog-posts) I expected like the below screenshot Screenshot from 2021-11-25 23-12-20. you can download and play with this [repo](https://designrevision.com/downloads/shards-dashboard-lite-react/).. Don't worry about the styling. It is already added in the assert file.

Hey,
The changes have been made as you said.
- Categories are merged and sorted based on the number of counts(Ascending).
- Each post will consist of a badge with the matching tag of it in the sorted list of tags.
- Badge's color will be differentiated based on the category.

File: src/components/Slider.js [hunk 1]
```diff
@@ -1,42 +1,76 @@
...
+
+        const tagArticle = [];
+        let removedBlogs = blogs;
+        for (let i = 0; i < sortedTagsArray.length; ++i) {
+          const blogsWithTag = removedBlogs.filter(blog => blog.categories.includes(sortedTagsArray[i])); //filter
+          removedBlogs = removedBlogs.filter(blog => blogsWithTag.indexOf(blog) == -1); //exclude
+          if (blogsWithTag.length > 0) {
+            blogsWithTag.forEach(item => {
+              item[`tag`] = sortedTagsArray[i];
+              item['tagNo'] = i + 1;
```
Comment (sabesansathananthan) [inline src/components/Slider.js:57]: Tag No is wrong Correct it

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]
requested change

File: src/components/Slider.js (hunk 1 above)
Comment (Mathursan-007) [reply src/components/Slider.js]: resolved

**Review by sabesansathananthan — APPROVED** [review]
Thank you for your PR

**Pull request merged**